datafusion/sql/src/statement.rs

```diff
@@ -1709,18 +1709,22 @@ impl<S: ContextProvider> SqlToRel<'_, S> {
         // Do a table lookup to verify the table exists
         let table_ref = self.object_name_to_table_reference(table_name.clone())?;
         let table_source = self.context_provider.get_table_source(table_ref.clone())?;
-        let schema = (*table_source.schema()).clone();
-        let schema = DFSchema::try_from(schema)?;
-        let scan =
-            LogicalPlanBuilder::scan(table_ref.clone(), table_source, None)?.build()?;
+        let schema = table_source.schema().to_dfschema_ref()?;
+        let scan = LogicalPlanBuilder::scan(
+            //object_name_to_string(&table_name),
```

Comment:
left over?

Reply:
it showed up after rebase, so i left it in case i need to go back to i. will be removed